core/core.cc

```diff
@@ -438,6 +518,49 @@ App::App(int argc, char **argv)
       graphs.push_back(graph);
       graph = default_graph();
     }
+
+    // -- add by Yuankun
```

Comment:
I think this is unnecessary with scratch_bytes.

Reply:
Thanks, fixed.